### DISCUSSION TITLE
How to access the Agent Grafana Flow UI when using the Helm Chart?

### DISCUSSION THREAD

**Discussion by danygielow** [grafana-cold-storage/agent#3360] [Q&A]
Hello, 

I use the Grafana Agent Helm Chart to deploy to Kubernetes.
The Ingress is only created for Faro, it seems.

How can I create an Ingress for the Grafana Agent Flow UI.

Chart Version used is 0.10.0

Thanks

**Comment by tpaschalis** [comment]
Hey there 👋 
Would you mind opening an issue for this? Also, PRs to the helm chart are welcome!
